packages/flame/lib/src/game/game_render_box.dart

```diff
@@ -60,6 +63,11 @@ class GameRenderBox extends RenderBox with WidgetsBindingObserver {
     markNeedsPaint();
   }
 
+  @override
+  void performLayout() {
+    size = constraints.biggest;
```

Comment:
how was this not a problem before? was it a bug?

Reply:
We have compute dry.

Reply:
That s called because we say to the framework that we are sized by the parent.